plugins/color_panel/gtk/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.10)
+set(PROJECT_NAME "color_panel")
+project(${PROJECT_NAME} LANGUAGES CXX)
+
+set(PLUGIN_NAME "${PROJECT_NAME}_plugin")
+
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(GTK REQUIRED IMPORTED_TARGET gtk+-3.0)
```

Comment:
Sorry, I think this comment was unclear. We should still be linking to GTK in the plugin build, since it does require GTK, I just wouldn't expect us to need these to lines creating the PkgConfig::GTK library to link to since the one from the main CMake file should be visible here.